apis/infrastructure/v1/nim_types.go

```diff
@@ -0,0 +1,12 @@
+package v1
+
+import (
+	operatorv1 "github.com/openshift/api/operator/v1"
+)
+
+// nimSpec enables NVIDIA NIM integration
+type NimSpec struct {
```

Comment:
since NIM is a kserve only capability, shouldn't it be only part of kserve ?

Reply:
let me move it to kserve_types.go

Reply:
updated